plugins/funind/indfun_common.ml

```diff
@@ -144,7 +144,9 @@ let save with_clean id const ?hook (locality,_,kind) =
 	(locality, ConstRef kn)
   in
   if with_clean then Proof_global.discard_current ();
-  Lemmas.call_hook ?hook ~fix_exn l r;
+  (* XXX: need help *)
+  let uctx = UState.empty in
```

Comment:
Make the `save` function take a ustate? Or make the caller (there's only 1) call the hook themselves (ie 6e8ee9cd4c3da1d743664c994f7c61d2009926c4)

Reply:
The whole `save` function should be indeed removed I think; but let's add the argument for now.